### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by spond**

### CODE REVIEW

File: src/core/tree.cpp [hunk 1]
```diff
@@ -2756,7 +2756,7 @@
     //printf ("%ld %d\n", nodesToDo.lLength, hasExpForm);
     //ObjectToConsole(&isExplicitForm);
     
-    unsigned long id;
+    //unsigned long id;
```
[the remark this answers is not in the document]
Comment (stevenweaver) [reply src/core/tree.cpp:2759]: This line should be removed.

**Review by stevenweaver — CHANGES_REQUESTED** [review]

**Comment by stevenweaver** [comment]
2.5.59 tag should be removed because it appears that this PR was intended to be merged into `master` beforehand.

File: res/TemplateBatchFiles/SelectionAnalyses/error-filter.bf [hunk 2]
```diff
@@ -175,6 +175,18 @@ for (p = 0; p < efilter.input[terms.json.partition_count]; p+=1) {
                         efilter.site_BF2 = 1e25;
                     }
                     
+
```
[the remark this answers is not in the document]
Comment (spond) [reply res/TemplateBatchFiles/SelectionAnalyses/error-filter.bf:178]: That's my bad habit of leaving debugging code as comments :(

[the remark this answers is not in the document]
Comment (spond) [reply res/TemplateBatchFiles/SelectionAnalyses/error-filter.bf]: Changes made as requested.

File: src/core/formula.cpp [hunk 3]
```diff
@@ -2309,15 +2309,34 @@ bool _Formula::ConvertToSimple (_AVLList& variable_index) {
...
             _Operation* this_op = ItemAt (i);
-            simpleExpressionStatus[i] = -4L;
+            simpleExpressionStatus[i] = -64L;
             if (this_op->theNumber) {
-                simpleExpressionStatus[i] = -1L;
+                if (used_float_slots < available_slots) {
+                    store_constant [used_float_slots++] = this_op->theNumber->Value();
+                    simpleExpressionStatus[i] = -1L - used_float_slots;
+                } else {
+                    /*if (used_float_slots == available_slots) {
```
[the remark this answers is not in the document]
Comment (spond) [reply src/core/formula.cpp]: Changes made as requested.

File: src/core/formula.cpp [hunk 4]
```diff
@@ -2389,99 +2410,112 @@ void _Formula::ConvertFromSimpleList (_SimpleList const& variableIndex) {
 }
 
 //__________________________________________________________________________________
-hyFloat _Formula::ComputeSimple (_SimpleFormulaDatum* stack, _SimpleFormulaDatum* varValues) {
+hyFloat _Formula::ComputeSimple (_SimpleFormulaDatum* stack, _SimpleFormulaDatum* varValues, long allocate_local) {
```
[the remark this answers is not in the document]
Comment (spond) [reply src/core/formula.cpp]: Changes made as requested.

**Comment by spond** [comment]
Change requests should be resolved.

**Review by stevenweaver — APPROVED** [review]

**Pull request merged**